index.d.ts

```diff
@@ -1,8 +1,8 @@
 /**
- * An *action* is a plain object that represents an intention to change the
+ * An *action* is a plain object that represents an intention to change
  * state. Actions are the only way to get data into the store. Any data,
  * whether from UI events, network callbacks, or other sources such as
- * WebSockets needs to eventually be dispatched as actions.
+ * WebSockets need to eventually be dispatched as actions.
```

Comment:
Data is plural; datum is singular. 

Adam

> On Apr 14, 2016, at 4:50 PM, Dan Abramov <notifications@github.com> wrote:
> 
> In index.d.ts:
> 
> >   * state. Actions are the only way to get data into the store. Any data,
> >   * whether from UI events, network callbacks, or other sources such as
> > - * WebSockets needs to eventually be dispatched as actions.
> > + * WebSockets need to eventually be dispatched as actions.
> To clarify, needs corresponds to data before. I think needs is correct here.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>